config/test/protractor.e2e.dist.conf.js

```diff
@@ -8,6 +8,17 @@ settings.specs = [
     '../../test/e2e/**/*.js',
 ];
 
+// Enable PhantomJS testing here.
+// Its seems that there are no random failures when testing with
+// non instrumented code
+// TODO: remove Phantomjs and use real browsers with Vagrant
+settings.capabilities = {
+    browserName: 'phantomjs',
+    //'phantomjs.binary.path': 'node_modules/.bin/phantomjs' + (process.platform === 'win32' ? '.cmd' : ''),
```

Comment:
It works. Just replaced the previous line for ```require('phantomjs').path ```, which resolves the binary path in both environments and seems more elegant